platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -47,15 +46,6 @@ const relativeDecoder = pipe(
 
 const mainDecoder = Decode.type({
```

Comment:
Can't comment above, since it's not changed, but it would probably be a bit clearer if thumbnail is renamed thumbnailPath. That would make the "relativeDecoder" a bit clearer, since what it does is actually resolve a path either locally or in a `document`-endowed environment.

> It could be renamed `relativePathDecoder`, too.